src/windows/webview.c

```diff
@@ -684,11 +702,15 @@ bool mty_webview_is_steam(void)
 
 bool mty_webview_is_available(void)
 {
-	HMODULE webview = webview_load_dll();
-	if (webview) {
-		FreeLibrary(webview);
-		return true;
-	}
+	WCHAR path[MTY_PATH_MAX] = {0};
 
-	return false;
+	bool have_path = webview_dll_path(path, false);
+	if (!have_path)
+		have_path = webview_dll_path(path, true);
+
+	if (!have_path)
+		return false;
+
+	// Loading the lib would be ideal to be sure, but repeated loads eventually cause issues
+	return PathFileExists(path);
```

Comment:
You should add to the comment exactly _what_ issues are caused due to repeated loads viz. windows not cleaning things up properly. Include a link to the MSDN article describing it as well.

Reply:
I don't recall if it was an MSDN article, and I'd need to find my source again, but that's reasonable